ztools/x509/pkix/pkix.go

```diff
@@ -208,3 +208,17 @@ type RevokedCertificate struct {
 	RevocationTime time.Time
 	Extensions     []Extension `asn1:"optional"`
 }
+
+// OtherName represents the ASN.1 structure of the same name. See RFC
+// 5280, section 4.2.1.6.
+type OtherName struct{
```

Comment:
Gotcha - so there is no need to assign `asn1:"tag:1,explicit"` here for `Value` because it is parsed that way by default?